packages/loop-harness/scripts/loop_scheduler.py

```
@@ -0,0 +1,501 @@
+        cwd=project_dir,
+        capture_output=True,
+        text=True,
+        timeout=30,
+        check=True,
+    )
+    return completed.stdout.strip()
+
+
+def _gh_list_issues(repo: str, label: str) -> str:
+    """Run `gh api .../issues` and return its raw stdout (isolated for test monkeypatching)."""
+    completed = subprocess.run(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)